src/bazel_version.rs

```diff
@@ -89,16 +89,16 @@ pub fn determine_bazelisk_version(path: &Path) -> Option<String> {
         return Some(version_str.trim().to_string());
     }
     let workspace_root = get_workspace_path(path)?;
-    if let Ok(bazeliskrc) = fs::read_to_string(workspace_root.join(".bazeliskrc")) {
+    if let Ok(bazelversion) = fs::read_to_string(workspace_root.join(".bazelversion")) {
+        return Some(bazelversion.trim().to_string());
```

Comment:
`cargo clippy` prefers this other, more instructional coding style. That's also why the Linux build is red

```suggestion
       Some(bazelversion.trim().to_string())
```

Reply:
although... I don't quite understand why `cargo clippy` was fine with the previous code in the first place. It should have also flagged the previous code. Weird...

Either way: Let's change this to make clippy happy